metriq_gym/run.py

```diff
@@ -61,9 +61,26 @@ def dispatch_job(args: argparse.Namespace, job_manager: JobManager) -> None:
     )
     logger.info(f"Job dispatched with ID: {job_id}")
 
-
 def poll_job(args: argparse.Namespace, job_manager: JobManager) -> None:
     logger.info("Polling job...")
```

Comment:
We want to show this message only after. It would be weird to see "Polling job..." and then "Select a job..."